008_team_6_agent_design/008_team_6_agent_design.tex

```diff
@@ -1 +1,1014 @@
+%%
+%% Strategy
+%%
+
+% \subsection{Conceptual Description}
+\subsection{Variation by Social Motive}
+With the agent's behavioural theory defined, its corresponding strategy can be expressed based on the defining qualities of its social motive. As the social motive varies from Altruist towards Narcissist, individual utility is valued increasingly more than collective utility, resulting in an increase in food consumption. Whilst social motives are defined using a continuous spectrum, the agent’s consumption and communication strategies are defined on bins along this spectrum, with one bin for each of the four discrete social motives. Thus, which strategy is carried out is decided in a switch-case statement on the agent's current behaviour.
+
+
+% \subsubsection{Altruist}
+% Since an altruistic agent is only concerned for the well-being of others with total disregard for itself, it takes no food and aims to maximise the amount of food left for the remaining agents.
+
+% \subsubsection{Collectivist}
+% A collectivist will consume the minimum food needed to survive. The result is that, on injection, this agent takes no food due to having high starting health. As turns progress, when the agent approaches a critical condition, it takes the minimal amount to satisfice itself. Since there is a time delay between when this agent first falls critical and is terminated, we assert that all collectivists stagger the day that they take food (randomised between 1 and the maximum number of days an agent can stay critical for) to ensure that all the available resources are not depleted at one given time.
+
+% \subsubsection{Selfish}
+% A selfish agent will consume enough food to be satiated and remain `healthy'. In this report, we define satiation to represent a health of at least 30\% of the maximum value.
+
+% \subsubsection{Narcissistic}
+% A narcissistic agent will consume the maximum amount of food possible to take at a given iteration, since it will purely be concerned for its well-being and be willing to harm other agents.
+
+
+\subsubsection{Altruist}
+An altruistic agent will always take 0 food, as it is only concerned for the well-being of others with a total disregard for itself.
+
+\subsubsection{Collectivist}
+A collectivist agent will consume just enough food to survive, and consume no food when not in danger of dying. Based on the implemented health function, the agent is only in danger of dying when in the Critical region, and will die if it spends \textit{N = maxCritical} days in this zone without consuming the minimum required units of food to return to the \textit{WeakLevel}. The collectivist agent, having entered the Critical region, will consume food on a randomly assigned day within the allowable period to leave the Critical region. The day of consumption is randomised to avoid the case of all agents needing to consume food on the same day, as there is not enough food on the platform to satisfice every agent in the tower at one time.
```

Comment:
```suggestion
A collectivist agent will consume just enough food to survive, and consume no food when not in danger of dying. Based on the implemented health function, the agent is only in danger of dying when in the Critical region, and will die if it spends \texttt{N = maxCritical} days in this zone without consuming the minimum required units of food to return to the \texttt{WeakLevel}. The collectivist agent, having entered the Critical region, will consume food on a randomly assigned day within the allowable period to leave the Critical region. The day of consumption is randomised to avoid the case of all agents needing to consume food on the same day, as there is not enough food on the platform to satisfice every agent in the tower at one time.
```